### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Review by davisagli — COMMENTED** [review]
@sneridagh Don't most of these new how-to docs belong in volto's docs rather than in volto-light-theme?

File: README.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ It aims to be future proof, so it has to be aligned with the upcoming Volto visi
 
 ![Volto-Light-Theme](https://github.com/kitconcept/volto-light-theme/raw/main/volto-light-theme.png)
 
+## Documentation
+
+You can find the documentation of this package in: https://volto-light-theme.readthedocs.io
```
Comment (davisagli) [inline README.md:22]: We should also remove some of the info below that is now included in the docs (but I guess only after the RTD site is fully deployed)

Comment (davisagli) [reply README.md:22]: For one thing, the Installation section below is not up to date with what you added in the new docs.

**Comment by sneridagh** [comment]
@davisagli I've amended the docs, so this PR should go first before #420

File: docs/install.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    "@kitconcept/volto-highlight-block": "^4.0.0",
+    "@kitconcept/volto-introduction-block": "^1.0.0",
+    "@kitconcept/volto-separator-block": "^4.1.2",
+    "@kitconcept/volto-slider-block": "^6.3.1",
+    "@kitconcept/volto-light-theme": "^5.0.1",
+  }
+```
+
+```{note}
+This versions are orientative, and up to date as per 2024-11-04.
```
Comment (davisagli) [inline docs/install.md:20]: I'm not sure what orientative means.

**Review by davisagli — APPROVED** [review]
@sneridagh Basically looks good to me.

File: docs/install.md (hunk 2 above)
Comment (stevepiercy) [reply docs/install.md:20]: I think he put it there by occident.

File: docs/compatibility.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "keywords": "Volto Light Theme, Compatibility"
+---
+
+# Compatibility
+
+| VLT version | Volto version |
+|-------------|---------------|
+| 3.x.x       |   >= Volto 17.0.0-alpha.16  |
+| 4.x.x       |   < Volto 17.18.0  |
+| 5.x.x       |   >= Volto 17.18.0 or >=Volto 18.0.0-alpha.36  |
```
Comment (stevepiercy) [inline docs/compatibility.md:16]: Should this be extended with v6?

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+version_from_package_json = json.loads(data)["version"]
+
+if version_from_package_json:
+    # The short X.Y version.
+    version = version_from_package_json
+    # The full version, including alpha/beta/rc tags.
+    release = version_from_package_json
+else:
+    version = "17.15.6"
+    release = "17.15.6"
```
Comment (stevepiercy) [inline docs/conf.py:45]: Verify that the version and release are correct.

File: docs/contributing.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+make start-test-acceptance-frontend-dev
+```
+
+In a second terminal session, start the backend acceptance server.
+
+```shell
+make start-test-acceptance-server
+```
+
+Start the Cypress interactive test runner.
```
Comment (stevepiercy) [inline docs/contributing.md:133]: In which terminal session? First or second?

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Review by stevepiercy — APPROVED** [review]
I've completed my review by pushing commits. I left a few comments for consideration. Otherwise it is good to merge, and we can check RTD to verify it works.

File: docs/install.md (hunk 2 above)
Comment (stevepiercy) [reply docs/install.md]: I rewrote this in https://github.com/kitconcept/volto-light-theme/pull/423/commits/0a6d4feecc6b66dc9c334a0db1c94b0bae4dade6

**Comment by sneridagh** [comment]
@stevepiercy thanks!!

**Comment by stevepiercy** [comment]
Let's verify that the home page URL is correct.

https://volto-light-theme.readthedocs.io/

https://volto-light-theme.readthedocs.io/sitemap.xml

https://volto-light-theme.readthedocs.io/robots.txt

Oopsie, one mistake. PR incoming. No changelog needed.

**Pull request merged**